packages/create-plugin/templates/app/provisioning/plugins/apps.yaml

```diff
@@ -2,4 +2,10 @@ apiVersion: 1
 
 apps:
   - type: '{{ normalize_id pluginName orgName 'app' }}'
+    org_id: 1
+    org_name: '{{ orgName }}'
```

Comment:
I'm not 💯  sure of this but I don't think we should be setting this to the orgName of the plugin. I don't know what impact it has but based on [this comment](https://grafana.com/docs/grafana/latest/administration/provisioning/#example-plugin-configuration-file) I think it relates to the orgs set in the grafana application:

![image](https://github.com/grafana/plugin-tools/assets/73201/f581fbed-a2fb-4ede-b7dc-fb01c0dcf9b4)

Reply:
I have created the app plugin and I see that it has orgName which I provided during the creation.